code/game/objects/structures/watercloset.dm

```diff
@@ -133,7 +133,7 @@
 
 /obj/structure/urinal/New()
 	..()
```

Comment:
Change this to Initialize() and make . = ..() it causes other issues by being /New()